### ISSUE TITLE
[feature request] Option to create links with target=_blank as function

### ISSUE THREAD

**Issue by karlguillotte** [jonschlinkert/remarkable#213]
#171 It is a great feature and it is working well! Thanks!

May I suggest an improvement? 

Perhaps, the linkTarget option could be a function as well. In that case, we would be able to test the URL and return an appropriate target value.

**Comment by Venryx** [comment]
Yeah, this would be nice for having the "target=_blank" only be applied for external links.
